snuba/snuba_migrations/functions_summaries/0001_functions_summaries_create_table.py

```diff
@@ -0,0 +1,81 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import UUID, Column, DateTime, String, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import OperationTarget, SqlOperation
+from snuba.utils.schemas import Float
+
+storage_set_name = StorageSetKey.FUNCTIONS_SUMMARIES
+local_table_name = "functions_summaries_local"
+dist_table_name = "functions_summaries_dist"
+
+columns: List[Column[Modifiers]] = [
+    Column("project_id", UInt(64)),
+    # profile_id is nullable since this will only be used by transaction-based profiling
+    Column("profile_id", UUID(Modifiers(nullable=True, low_cardinality=False))),
+    # profiler_id is nullable since this will only be used by continuous profiling
+    Column("profiler_id", UUID(Modifiers(nullable=True, low_cardinality=False))),
```

Comment:
A nullable is a bit like storing 2 columns where one is a boolean `is_null`. Maybe we can have 1 column with the UUID and a boolean `is_continuous` instead.